docs/General/manageDevice.md

```diff
@@ -16,11 +16,15 @@
     [![mps](../assets/images/MPS_ManageDevice.png)](../assets/images/MPS_ManageDevice.png)
 
     !!! note
-        If activated into Client Control Mode(CCM), the KVM and SOL features will not function in the current release. To use KVM/SOL, follow the [ACM Activation Tutorial](createProfileACM.md) to see how to configure a device into Admin Control Mode.
+        If activated into Client Control Mode(CCM), the KVM and SOL features will require a user consent code to be entered that will be displayed on the device. To use KVM/SOL without user consent, follow the [ACM Activation Tutorial](createProfileACM.md) for how to configure a device into Admin Control Mode.
```

Comment:
If activated in Client Control Mode(CCM), the KVM and SOL features require entering a user consent code, which will be displayed on the device. To use KVM/SOL without user consent, follow the [ACM Activation Tutorial](createProfileACM.md) for how to configure a device into Admin Control Mode.